docs/contributing/forks.md

```diff
@@ -0,0 +1,27 @@
+# Accepting PRs from a fork
+
+Unfortunately, our CI does not work with PRs from forks. This is because we pull in some proprietary executables and CI will only have the right secrets to download those executables if it is run from the main repo.
+
+To accept a PR from a fork, you create a new branch and cherry pick the commits from the forked PR onto the new branch. Then you push that branch to the main repo and create a new PR.
+
+In order to cherry pick the commits, you will have to [add the fork as a remote to your local repo](https://simonhartcher.com/how-to-cherry-pick-a-git-commit-from-a-fork/), fetch the remote and then cherry-pick.
```

Comment:
It becomes more obvious the further one reads, but please consider putting a thing in here that says something like "for fossa employees" to make it explicit

Reply:
Good point, especially as this is in the "contributing" directory. Done